src/cbmc/cbmc_parse_options.cpp

```diff
@@ -430,6 +434,80 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
 
 /*******************************************************************\
 
+Function: cbmc_parse_optionst::unwind_enum_static
```

Comment:
That's not the ideal place for this code. 
Can you move this into goto-programs/ in the style of the remove_* passes there?